CHANGES/7394.bugfix

```diff
@@ -0,0 +1,2 @@
+Matched DistributionTree serializers with the models
```

Comment:
Please make it more user-focused, they are not supposed to know about serializers.